frontend/src/features/permits/components/permit-list/ApplicationsInProgressList.tsx

```diff
@@ -88,6 +89,8 @@ export const ApplicationsInProgressList = () => {
   const [isDeleteDialogOpen, setIsDeleteDialogOpen] = useState(false);
   const [rowSelection, setRowSelection] = useState<RowSelectionState>({});
   const hasNoRowsSelected = Object.keys(rowSelection).length === 0;
+  const featureFlags = useFeatureFlagsQuery();
+  console.log('feature flags data', featureFlags?.data);
```

Comment:
WILL BE removing these 2 lines - just kept them in for now as proof that the API call and hook are working